commands/MakeCommand.js

```
@@ -81,7 +78,7 @@ export class MakeCommand extends Commander {
     return {
       ok: true,
       exitCode,
-      message: output + exitMsg,
+      display: output + exitMsg,
```

Biome (2.4.15)

[error] This property is later overwritten by an object member with the same name.

(lint/suspicious/noDuplicateObjectKeys)